vision.py

```diff
@@ -10,15 +11,72 @@
 from tqdm import tqdm
 from PIL import Image
 import tracking
+import time
```

Comment:
Please add standard library imports to the top import block, and third party imports to the second import block.

First party imports (i.e. from this project) should go into a third block.